extension.js

```diff
@@ -66,6 +66,8 @@ var global_state = null;
 
 var preview_panel = null;
 
+var comment_symbol ='#';
```

Comment:
need extra whitespace!

Reply:
Ok, no problem, I can fix this. Perhaps there should be some kind of linter configured, so this does not happen when coding. I really tried to adapt to the coding style. However, it took me half an hour to code and test the enhancement. It took me another hour to get all the trailing white space back again (see e.g. line 291, 410, 415, 528, ...) because my git and editor would remove it all the time and then the pull request contains many unrelated changes. This is also true for the README file.

Reply:
I am sorry for your troubles. But you could have fixed the trailing whitespaces and I would gladly accept that ;) BTW I think I have a linter config in the repo although I am not sure if it detects trailing spaces.
And trailing spaces in README is a feature, not a bug, otherwise github would concatenate different lines into a single long continuous paragraph - a very annoying behavior.